packages/utils/src/url.ts

```diff
@@ -70,8 +70,8 @@ export function getSanitizedUrlString(url: PartialURL): string {
         // Always filter out authority
         .replace(/^.*@/, '[filtered]:[filtered]@')
         // Don't show standard :80 (http) and :443 (https) ports to reduce the noise
-        .replace(':80', '')
-        .replace(':443', '')) ||
+        .replace(/(:80)$/, '')
+        .replace(/(:443)$/, '')) ||
```

Comment:
Why not just check for port, and call the port setter?

Reply:
what do you mean by port setter?

also feel free to completely tear apart all of this URL related code - you're def the expert here for this 😄

Reply:
Eventually, I think we should convert this function to use `new URL` if the global exists. But I don't think we should do it as part of this PR. Can you add TODO to the code?

Reply:
will add todo!